src/main/java/org/jitsi/videobridge/rest/root/colibri/debug/Debug.java

```diff
@@ -63,6 +65,32 @@ public Response enableFeature(@PathParam("feature") DebugFeatures feature)
         return Response.ok().build();
     }
 
+    @POST
+    @Path("/{confId}/{epId}/enable/{feature}")
+    @Consumes(MediaType.APPLICATION_JSON)
+    public Response enableEndpointFeature(
```

Comment:
We'll want one to disable as well I think, yeah?

Reply:
yes! shall i split the method or this approach is fine?

Reply:
I think this is fine.